### PR TITLE
Initial commit of rubocop.yml and README updates

### PR DESCRIPTION
**Pull request by charlesgwilson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2330,50 +2330,6 @@ condition](#safe-assignment-in-condition).
     SOME_CONST = 5
     ```
 
-  * <a name="bool-methods-qmark"></a>
```
Comment (rayfaddis) [inline README.md]: Why nuke this section?

Comment (charlesgwilson) [reply README.md]: Do you have a recommendation on how you'd like it presented? was nuking it to reflect the fact we disable this cop (auxiliary verbs are acceptable was the recommendation), not just tweak the settings. Im all for not nuking if you have thoughts on how to present it better or if you disagree with the recommendation of the auxiliary verbs

Comment (rayfaddis) [reply README.md]: I agree exactly how they spell it out: predicate should end in question mark and should not start with is, does, can.

Comment (charlesgwilson) [reply README.md]: I agree all predicate methods should end in a question mark. The issue with the auxiliary verbs is that we use them in permissions related predicate methods, ie can_update, can_view, etc, so are you recommending we change this practice?

Comment (rayfaddis) [reply README.md]: Darn it, on those permissions ones I like the can_update can_views actually as is haha. Though it is definitely more Rails-ish to change them to something else like `update?` (if even possible) or better `update_allowed_by?` and so on `view_allowed_by?`

**Pull request merged**